source/extensions/filters/http/ext_proc/ext_proc.h

```diff
@@ -114,6 +125,8 @@ class FilterConfig {
   ExtProcFilterStats stats_;
   const envoy::extensions::filters::http::ext_proc::v3::ProcessingMode processing_mode_;
   const Filters::Common::MutationRules::Checker mutation_checker_;
+  const std::string filter_state_name_;
+  const envoy::config::core::v3::Metadata metadata_;
```

Comment:
I think you need to implement the serializers to make this work with access logs `%FILTER_STATE(KEY:F):Z% - that's required for interop with Otel, for example. Related to this, we probably need a structured lookup for fields in filter state objects (https://github.com/envoyproxy/envoy/issues/24013), since metadata looks rather verbose.

Reply:
but is this serializer mandatory for this PR? custom access log filters can consume this with no issue right? Thats the prime use case here. I can remove the doc comment about printing this on file access log using command operators

Reply:
Not required - but we want the standard loggers to be on feature parity with custom loggers. Can be done later.